src/base/QXmppJingleIq.cpp

```diff
@@ -839,24 +851,37 @@ void QXmppJingleIq::setResponder(const QString &responder)
     d->responder = responder;
 }
 
+#if QXMPP_DEPRECATED_SINCE(1, 5)
```

Comment:
not in the cpp file please, the function needs to be compiled in for users needing it